en/drafts/originals/image-classification-neural-networks.md

```diff
@@ -25,150 +25,154 @@ doi: tbc
+How exactly do artificial neurons work? Rather than diving directly into training them, it is helpful to first gain a broader understanding of their infrastructure. Let's say we are interested in a simple task such as determining if an image is a picture of a square or triangle. If you have done any kind of coding, you will know that most programs solve this by processing a sequence of steps. Loops and statements (such as `for`, `while` and `if`) allow our program to have branches that simulate logical thinking. In the case of determining whether an image contains a shape, we could code our program to count the number of sides and display "square" if it finds four, or "triangle" if it finds three. Distinguishing between geometric objects may seem like a relatively simple task, but it requires a programmer to not only define a shape's characteristics but also implement logic that discerns those characteristics. This becomes increasingly difficult as we run into scenarios where distinctions between images are more complex. For instance, look at the following images:
 
-{% include figure.html filename="cat.jpg" caption="Figure 1. A picture of a cat" %}
-{% include figure.html filename="dog.jpg" caption="Figure 2. A picture of a dog" %}
+{% include figure.html filename="cat.jpg" alt="Visual description of figure image" caption="Figure 1. A picture of a cat" %}
+{% include figure.html filename="dog.jpg" alt="Visual description of figure image" caption="Figure 2. A picture of a dog" %}
 
 As humans, we can easily determine which one is a dog and which one is a cat. However, outlining the exact differences proves challenging. It turns out that humans are usually a lot better at handling the nuances of vision than computers. What if we could get a computer to process images the way our brains do? This question and insight forms the basis of artificial neurons.
 
-As the name implies, artificial neurons take their inspiration from neurons in the brain. The following is a simplified look at what a biological and artificial neuron looks like:
+As the name implies, artificial neurons take their inspiration from neurons in the brain. The following is a simplified diagram[^3] of how a biological and artificial neuron work:
 
-{% include figure.html filename="neuron.png" caption="Figure 3. A diagram of a biological and an artificial neuron. https://www.oreilly.com/library/view/mobile-artificial-intelligence/9781789344073/5ac86b1a-c080-49ea-a234-5335f12f15af.xhtml" %}
+{% include figure.html filename="neuron.png" alt="Visual description of figure image" caption="Figure 3. A diagram of a biological and an artificial neuron." %}
```

Comment:
There is no Figure 4 or 5 ? 

We move directly from Figure 3 at l.65 to Figure 6 at l.123.

Are they missing? Or do we need to re-number and re-name remaining images?

Reply:
Hmm...this may be due to something I deleted. I will double check real quick.